packages/api/src/test-helpers.ts

```diff
@@ -238,3 +240,21 @@ export async function setupUsers(
     nonAdminApiKey,
   };
 }
+
+export async function resumeTestTusUpload(upload: tus.Upload) {
+  upload.findPreviousUploads().then(function (previousUploads) {
+    if (previousUploads.length) {
+      upload.resumeFromPreviousUpload(previousUploads[0]);
+    }
+    upload.start();
+  });
```

Comment:
Since the function is async you can write this more readable with async/await.
```suggestion
  const previousUploads = await upload.findPreviousUploads()
  if (previousUploads.length) {
    upload.resumeFromPreviousUpload(previousUploads[0]);
  }
  return upload.start(); // (just in case it returns something? if it doesn't, can avoid the `return`)
```